webapp/channels/src/components/admin_console/data_retention_settings/global_policy_form/index.ts

```diff
@@ -12,16 +12,29 @@ import {
     updateConfig,
 } from 'mattermost-redux/actions/admin';
 import type {GenericAction, ActionFunc} from 'mattermost-redux/types/actions';
+import {getDataRetentionTimeInHours} from 'mattermost-redux/utils/helpers';
 
 import {setNavigationBlocked} from 'actions/admin_actions.jsx';
 
+import type {GlobalState} from 'types/store';
+
 import GlobalPolicyForm from './global_policy_form';
 
 type Actions = {
     updateConfig: (config: Record<string, any>) => Promise<{ data?: AdminConfig; error?: ServerError }>;
     setNavigationBlocked: (blocked: boolean) => void;
 };
 
+function mapStateToProps(state: GlobalState) {
+    const messageRetentionHours = getDataRetentionTimeInHours(state.entities.admin.config.DataRetentionSettings?.MessageRetentionDays, state.entities.admin.config.DataRetentionSettings?.MessageRetentionHours);
+    const fileRetentionHours = getDataRetentionTimeInHours(state.entities.admin.config.DataRetentionSettings?.MessageRetentionDays, state.entities.admin.config.DataRetentionSettings?.MessageRetentionHours);
```

Comment:
You used the above `message` value into the `file` variable. `MessageRetentionDays` and `MessageRetentionHours`.

Reply:
Good catch